guides-and-tutorials/running-a-signer/how-to-monitor-signer.md

```diff
@@ -0,0 +1,165 @@
+# How to Monitor a Signer
+
+We will use [Grafana Cloud](https://grafana.com) to observe and monitor both the
+Signer and its corresponding Stacks node.
+
+## Requirements
+
+Grafana's application observability docs have a [great
+quick-start](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/). We will use:
+
+- Grafana Cloud to collect metrics and visualize them.
+- Grafana Alloy, on the Signer host, to push the metrics.
+
+### Creating a Grafana Cloud account
+
+Before we begin, create a [Grafana
+Cloud](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/grafana-cloud/) account (they offer a free tier that you can use).
+
+Once done, access your dashboard, click on "Connections", "Add new connection",
+and select "Hosted Prometheus metrics". Select "Via Grafana Alloy", then on step
+2 select "Run Grafana Alloy" to generate an API token. Note the token
+`GCLOUD_RW_API_KEY` and the params `GCLOUD_HOSTED_METRICS_URL` and
+`GCLOUD_HOSTED_METRICS_ID`, we will use them later.
+
+### Configuring the Signer and the Stacks node
+
+Ensure both your Signer configuration and your node configuration include the
+following lines:
+
+```toml
+# signer-config.toml
+# ...
+metrics_endpoint = "127.0.0.1:30001"
+```
+
+```toml
+# node-config.toml
+# ...
+prometheus_bind = "127.0.0.1:9153"
+```
+
+If you compile the application binaries yourself, remember to enable the Cargo
+feature `monitoring_prom` while building them, for example:
+
+```bash
+cargo build --features monitoring_prom,slog_json --release
+```
+
+Once both binaries are running with the updated configuration, you can peek
+at the metrics being exposed:
+
+```bash
+curl 127.0.0.1:30001/metrics
+
+# HELP stacks_signer_current_reward_cycle The current reward cycle
+# TYPE stacks_signer_current_reward_cycle gauge
+stacks_signer_current_reward_cycle 95
+# HELP stacks_signer_node_rpc_call_latencies_histogram Time (seconds) measuring round-trip RPC call latency to the Stacks node
+# TYPE stacks_signer_node_rpc_call_latencies_histogram histogram
+# ...
```

Comment:
nit: i would remove the comment for the ellipsis, since it's not part of the prom output (same for the trailing ellipsis)

Reply:
Thanks, addressed in ac4d1cfa9